src/main/resources/languages/en/messages.yml

```
@@ -39,6 +39,10 @@ commands:
     running: '&7Running debug mode with test: &6%test%'
     disabled: '&7Disabled debug mode.'
 
+  dump:
+    description: 'Contains commands to dump data to files'
```

Might want to specify that it is a dev/debug tool
```suggestion
    description: 'Debugging tool to dump data to files'
```